include/onnxruntime/core/graph/graph.h

```diff
@@ -1129,12 +1120,13 @@ class Graph {
   */
   Status InlineFunction(Node& node);
 
-  /** Initialize function body for the given node */
-  void InitFunctionBodyForNode(Node& node);
-
   /** Gets Model local functions from the root/parent graph.*/
   const std::unordered_map<std::string, const ONNX_NAMESPACE::FunctionProto*>& GetModelLocalFunctions() const;
 
+  const std::unordered_map<std::string, FunctionTemplate*>& GetModelLocalFunctionTemplates() const;
+
+  void InitModelLocalFunctionTemplatesMap(const std::vector<FunctionTemplate*>& model_function_templates);
```

Comment:
>onst std::vector<FunctionTemplate*> [](http://example.com/codeflow?start=43&length=35)

For const inputs, use gsl::span<FunctionTemplate* const> by value. This way, it can accept different containers. See coding standards.